src/sentry/utils/suspect_resolutions/commit_correlation.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Set
+
+from django.db.models import DateTimeField
+
+from sentry.models import CommitFileChange, GroupRelease, ReleaseCommit
+
+
+def is_issue_commit_correlated(resolved_issue_id: int, candidate_issue_id: int) -> bool:
+    first_released = (
+        GroupRelease.objects.filter(group_id=resolved_issue_id)
+        .values_list("first_seen", flat=True)
+        .first()
+    )
+    last_released = (
+        GroupRelease.objects.filter(group_id=resolved_issue_id)
+        .values_list("last_seen", flat=True)
+        .last()
+    )
```

Comment:
Wait, is this what we really want to do? Query for the first and last release for the resolved issue? 

I thought we wanted to pick a time-window [1-hour-before-resolved-time, resolved-time] to query for any another releases and check if they're correlated?

1-hour is fairly arbitrary in this case; it could be 2 or 3 hours. Consider making it an optional param in the function.

Reply:
The timestamps associated with the first and last release are being passed to `get_files_changed` as the start and end time windows, so we're getting all the files changed within each release associated with the issue